Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+
+ALCARECOPPSCalMaxTracksFilterExpress = hlt.hltHighLevel.clone(
+  TriggerResultsTag = cms.InputTag("TriggerResults","","HLTX"),
```

Comment:
```suggestion
  TriggerResultsTag = ("TriggerResults","","HLTX"),
```
By the way: are these configs intended to be used only in tests? I.e., is the process name "HLTX" stated inside these configurations (and not in the tests themselves) justified?

Reply:
This file `Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksExpress_cff.py` is not for test, it seems to be a production version. 

Can you suggest the preferred way of configuring `hltHighLevel` module, if the process name should be avoided?

Reply:
As far as I can see the process name "HLTX" is used in a test script that you define in this PR
If this config (and the other ones here) are used in production, I suspect that you shouldn't redefine the process name here, but only insed the tests themselves. But certainly @cms-sw/alca-l2 knows better about these configs, and can either confirm that this is correct, or suggest a better way to do it

Reply:
I've moved that part to the customisation part of testing script, see https://github.com/cms-sw/cmssw/pull/36273/commits/18c83a8f88a9ba2168b0438515eb0d211ecdd53c